site/docs/.vuepress/config.ts

```diff
@@ -700,7 +1032,50 @@ export default defineUserConfig<DefaultThemeOptions>({
         apiKey: "17b3527aa6f36e8d3fe2276b0f4d9633",
         indexName: "grammy",
         placeholder: "Search",
-        locales: { "/zh/": { placeholder: "搜索文档" } },
+        locales: {
+          "/zh/": { placeholder: "搜索文档" },
+          "/uz/": {
+            placeholder: "Qidirish",
+            translations: {
+              button: { buttonText: "Qidirish" },
+              modal: {
+                searchBox: {
+                  resetButtonTitle: "So'rovni tozalash",
+                  resetButtonAriaLabel: "So'rovni tozalash",
+                  cancelButtonText: "Bekor qilish",
+                  cancelButtonAriaLabel: "Bekor qilish",
+                },
+                startScreen: {
+                  recentSearchesTitle: "Yaqinda amalga oshirilgan qidiruvlar",
+                  noRecentSearchesText:
+                    "Yaqinda amalga oshirilgan qidiruvlar mavjud emas",
+                  saveRecentSearchButtonTitle: "Qidiruvni saqlash",
+                  removeRecentSearchButtonTitle: "Qidiruvni tarixdan o'chirish",
+                  favoriteSearchesTitle: "Sevimli qidiruvlar",
+                  removeFavoriteSearchButtonTitle:
+                    "Qidiruvni sevimlilar ro'yhatidan o'chirish",
+                },
+                errorScreen: {
+                  titleText: "Natijalarni qabul qilishda xatolik",
+                  helpText: "Iltimos, tarmoqga ulanganligingizni tekshiring.",
+                },
+                footer: {
+                  selectText: "tanlash",
+                  navigateText: "navigatsiya",
+                  closeText: "yopish",
+                  searchByText: "Qidiruv tizimi",
+                },
+                noResultsScreen: {
+                  noResultsText: "So'rov bo'yicha natija mavjud emas",
+                  suggestedQueryText: "Quyidagini qidirishga urinib ko'ring",
+                  openIssueText:
+                    "Bu so'rovdan natija chiqishiga ishonchingiz komilmi?",
+                  openIssueLinkText: "Bizga xabar bering",
+                },
+              },
+            },
+          },
```

Comment:
@WingLim @linbuxiao @taotie111 @JiquanWang99 could we also have these translations for Chinese (in a second PR)? The concern the Algolia search interface. We missed this in the early days when we were translating the config.

Reply:
cc @kidonng

Reply:
@IamItachiUchiha @mcpeblocker @sad1go0 could you leave a link here which points to the English originals of this configuration that you used when translating this? Such as the Algolia API reference or something.

Reply:
Here:
https://docsearch.algolia.com/docs/api/#translations